app.js

```diff
@@ -0,0 +1,18 @@
+const express = require("express");
+const app = express();
+const { getCategories } = require("./controller/games.controller.js");
+
+app.use(express.json());
+
```

Comment:
line 5 is not needed for GET requests. remove it.  only add code which is needed for a ticket.

Reply:
removed this as well